purchases/src/main/kotlin/com/revenuecat/purchases/common/events/EventRequest.kt

```diff
@@ -0,0 +1,29 @@
+package com.revenuecat.purchases.common.events
+
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.modules.SerializersModule
+import kotlinx.serialization.modules.polymorphic
+
+@Serializable
+data class EventRequest internal constructor(
+    internal val events: List<BackendEvent>,
+) {
+
+    companion object {
+        val json = Json {
+            serializersModule = SerializersModule {
+                polymorphic(BackendEvent::class) {
+                    subclass(BackendEvent.CustomerCenter::class, BackendEvent.CustomerCenter.serializer())
+                    subclass(BackendEvent.Paywalls::class, BackendEvent.Paywalls.serializer())
+                }
+            }
+            classDiscriminator = "discriminator"
```

Comment:
I thought in the request there's no discriminator?

Reply:
I ended up adding it because tests were failing. If I understood correctly, you must use a `classDiscriminator` when using polymorphic